EventVisualisation/Workflow/include/EveWorkflow/AO2DConverter.h

```diff
@@ -38,14 +38,14 @@ namespace o2::event_visualisation
 class TPCFastTransform;
 
 struct AO2DConverter {
-  o2::framework::Configurable<std::string> jsonPath{"jsons-folder", "./jsons", "name of the folder to store json files"};
+  o2::framework::Configurable<std::string> jsonPath{"jsons-folder", "./json", "name of the folder to store json files"};
```

Comment:
actually, also the directory on /home/epn/ where the jsons are stored is `jsons`.

Reply:
I changed this to follow the default directory from which ED reads them:

https://github.com/AliceO2Group/AliceO2/blob/191fc132d064ed8bfc256c9024bb2ed4eca1f95f/EventVisualisation/View/src/Options.cxx#L59

In any case I think the best solution would be to extract this value to a header, so it is in sync in both ED and the AOD workflow.